.codesandbox/ci.json

```
@@ -0,0 +1,7 @@
+{
+  "packages": ["packages/*-recommendations"],
```

We'll release all packages from that repo.

```suggestion
  "packages": ["packages/*"],
```